[BUG] PgVector index name uses full table name even if schema based

When using an index with PgVector (or hybrid search of that matter), the code munges the full table name along with the index suffix:

```
if (searchMode == SearchMode.HYBRID) {
                String ftsIndexName = table + "_text_fts_gin_index";
                query = String.format(
                        "CREATE INDEX IF NOT EXISTS %s ON %s " + "USING gin (to_tsvector('%s', coalesce(text, '')))",
                        ftsIndexName, table, textSearchConfig);
                statement.executeUpdate(query);
            }
            if (useIndex) {
                final String indexName = table + "_ivfflat_index";
                query = String.format(
                        "CREATE INDEX IF NOT EXISTS %s ON %s " + "USING ivfflat (embedding vector_cosine_ops) "
                                + "WITH (lists = %s)",
                        indexName, table, ensureGreaterThanZero(indexListSize, "indexListSize"));
                statement.executeUpdate(query);
            }
```

This causes problems if tables are schema based (doc.document_embedding) since this would mean the index name then becomes `doc.document_embedding_text_fts_gin_index` which contains a prohibited character for index names (dot). 

**Describe the bug**
Functional bug. Schemas should be trimmed out (if provided)

**Log and Stack trace**

```
Caused by: java.lang.RuntimeException: Failed to execute 'CREATE INDEX IF NOT EXISTS doc.document_embedding_text_fts_gin_index ON doc.document_embedding USING gin (to_tsvector('simple', coalesce(text, '')))'
	at dev.langchain4j.store.embedding.pgvector.PgVectorEmbeddingStore.initTable(PgVectorEmbeddingStore.java:322)
	at dev.langchain4j.store.embedding.pgvector.PgVectorEmbeddingStore.<init>(PgVectorEmbeddingStore.java:141)
	at dev.langchain4j.store.embedding.pgvector.PgVectorEmbeddingStore$DatasourceBuilder.build(PgVectorEmbeddingStore.java:738)
```

```
Caused by: org.postgresql.util.PSQLException: ERROR: syntax error at or near "."
  Position: 31
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2676)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2366)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:356)
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:496)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:413)
	at org.postgresql.jdbc.PgStatement.executeWithFlags(PgStatement.java:333)
	at org.postgresql.jdbc.PgStatement.executeCachedSql(PgStatement.java:319)
	at org.postgresql.jdbc.PgStatement.executeWithFlags(PgStatement.java:295)
```

**To Reproduce**

```
PgVectorEmbeddingStore store = PgVectorEmbeddingStore.datasourceBuilder()  .table("doc.document_embedding").dimension(3072).searchMode(PgVectorEmbeddingStore.SearchMode.HYBRID).build();
```

**Expected behavior**

When creating the indices, the schema name should be stripped out and the plain table name should be used before munging with the index name

**Please complete the following information:**
- LangChain4j version: 1.11.0
- LLM(s) used: N/A
- Java version: 17
- Spring Boot version (if applicable):  N/A

**Additional context**

None

--- a/langchain4j-pgvector/src/main/java/dev/langchain4j/store/embedding/pgvector/PgVectorEmbeddingStore.java
+++ b/langchain4j-pgvector/src/main/java/dev/langchain4j/store/embedding/pgvector/PgVectorEmbeddingStore.java
@@ -333,16 +333,16 @@ public class PgVectorEmbeddingStore implements EmbeddingStore<TextSegment> {
                 statement.executeUpdate(query);
                 metadataHandler.createMetadataIndexes(statement, table);
             }
-
+            String cleanTableName = computeCleanTableName();
             if (searchMode == SearchMode.HYBRID) {
-                String ftsIndexName = table + "_text_fts_gin_index";
+                String ftsIndexName = cleanTableName + "_text_fts_gin_index";
                 query = String.format(
                         "CREATE INDEX IF NOT EXISTS %s ON %s " + "USING gin (to_tsvector('%s', coalesce(text, '')))",
                         ftsIndexName, table, textSearchConfig);
                 statement.executeUpdate(query);
             }
             if (useIndex) {
-                final String indexName = table + "_ivfflat_index";
+                final String indexName = cleanTableName + "_ivfflat_index";
                 query = String.format(
                         "CREATE INDEX IF NOT EXISTS %s ON %s " + "USING ivfflat (embedding vector_cosine_ops) "
                                 + "WITH (lists = %s)",
@@ -352,6 +352,11 @@ public class PgVectorEmbeddingStore implements EmbeddingStore<TextSegment> {
         } catch (SQLException e) {
             throw new RuntimeException(String.format("Failed to execute '%s'", query), e);
         }
+    }
+
+    private String computeCleanTableName() {
+        int lastDotIndex = table.lastIndexOf('.');
+        return lastDotIndex >= 0 ? table.substring(lastDotIndex + 1) : table;
     }
 
     /**

